apps/deeplearning_apps/style_transfer/app.py

```diff
@@ -0,0 +1,218 @@
+from html_content import html
+import logging
+
+from h2o_wave import Q, main, app, handle_on, on, ui
```

Comment:
Do not import `ui` here.
All ui creation should go into `cards.py`. Check template.